src/components/Slider/SliderBase/SliderBase.scss

```diff
@@ -0,0 +1,199 @@
+  &__base {
+    display: flex;
+    touch-action: none;
+    -webkit-tap-highlight-color: transparent;
+    width: 100%;
+  }
+
+  &__rail {
+    cursor: pointer;
+    display: inline-block;
+    position: relative;
+    touch-action: none;
+    -webkit-tap-highlight-color: transparent;
+    width: 100%;
+  }
+
+  &__track {
+    position: absolute;
+    border-radius: inherit;
+    @include theme-prop(background-color, primary-selected-color);
+    height: inherit;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 100%;
+  }
+
+  &__filled-track {
+    position: absolute;
+    border-radius: inherit;
+    @include theme-prop(background-color, primary-color);
+    height: inherit;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 100%;
+  }
+
+  &__thumb {
+    position: absolute;
+    border: 3px solid;
+    border-radius: 50%;
+    @include theme-prop(border-color, primary-color);
+    @include theme-prop(background-color, primary-background-color);
+    font-size: 14px;
+    top: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  &__thumb-label {
+    bottom: calc(100% + 4px);
+    left: 50%;
+    max-width: 50px;
+    overflow: hidden;
+    position: absolute;
+    text-overflow: ellipsis;
+    transform: translate(-50%, 0);
+    white-space: nowrap;
+  }
+
+  // thumb (for NOT-disabled)
+  &:not(&--disabled) {
+    .monday-slider {
+      &__thumb {
+        &:focus-visible {
+          @include theme-prop(border-color, primary-hover-color);
+          outline: 3px solid;
+          @include theme-prop(outline-color, primary-selected-color);
+          transform: translate(-50%, -50%) scale(1.33, 1.33);
+          transform-origin: center center;
+          transition: transform 0.2s $expand-animation-timing;
+        }
+        &:hover {
+          @include theme-prop(border-color, primary-hover-color);
+          cursor: grab;
+        }
+        &:active {
+          cursor: grabbing;
+          transform: translate(-50%, -50%) scale(1.33, 1.33);
+          transform-origin: center center;
+          transition: transform 0.2s $expand-animation-timing;
+        }
+      }
+    }
+  }
+
+  // modifications
+
+  &__base {
+    &--disabled {
+      * {
+        pointer-events: none;
+      }
+      cursor: not-allowed !important;
+      opacity: 0.4;
+    }
+
+    &--small {
+      padding: 0 8px;
+      .monday-slider {
+        &__rail {
+          border-radius: 2px;
+          height: 2px;
+          padding: 8px 0;
+        }
+        &__thumb {
+          height: 16px;
+          width: 16px;
+        }
+      }
+    }
+
+    &--medium {
+      padding: 0 12px;
+      .monday-slider {
+        &__rail {
+          border-radius: 2px;
+          height: 4px;
+          padding: 12px 0;
+        }
+        &__thumb {
+          height: 24px;
+          width: 24px;
+        }
+      }
+    }
+
+    &--large {
+      padding: 0 12px;
+      .monday-slider {
+        &__rail {
+          border-radius: 8px;
```

Comment:
we have border radius variables for few fixed border radius sizes

Reply:
For border radius we have 4 different patterns of usage:
![Screen Shot 2022-01-05 at 13 58 04](https://user-images.githubusercontent.com/96241324/148214159-2a674259-fb70-4b27-9833-65c0492bc24c.png)
Probably hardcoded and SASS vars are deprecated.
But here I have a couple of questions:
1. What to use direct Custom CSS property: `var(--border-radius-small)` or helper `getCSSVar("border-radius-small"),`
2. In Figma requirements we have 2px border-radius - but at  [UI Styles](https://github.com/mondaycom/monday-ui-style/blob/master/src/border-radius.scss) we only have 4-8-16 - probably we need to add extra-small options

Waiting you answer and I'll continue with this issue :)

Reply:
regarding the first question - the most updated best practice / guideline we recommend is to use the var directly: var(--border-radius-small)

regarding the second question - if Evigny thinks we need to add the extra small unit so let's add it :)